bedrock/privacy/templates/privacy/base-quantum.html

```diff
@@ -9,10 +9,10 @@
 {% endblock %}
 
 {% block page_css %}
-  {% stylesheet 'firefox_privacy' %}
+  {% stylesheet 'privacy_quantum' %}
 {% endblock %}
 
-{% block body_id %}firefox-notice{% endblock %}
+{% block body_class %}privacy-policy {% if switch('firefox-57-release') %}has-quantum{% endif %}{% endblock %}
```

Comment:
Just so we get an automatic logo update when Quantum launches. (Note to self: clean this up later.)